roles/nfs/upgrade/tasks/install_repository.yml

```diff
@@ -0,0 +1,160 @@
+---
+- name: Initialize
+  set_fact:
+   gpgcheck: "yes"
+
+- name: Disable gpg check
+  set_fact:
+   gpgcheck: "no"
+  when: scale_version < '5.0.4.0'
+
+- name: upgrade | nfs path
+  set_fact:
+   scale_nfs_url: 'ganesha_rpms/rhel7/'
+  when: ansible_distribution in scale_rhel_distribution and ansible_distribution_major_version == '7'
+
+- name: upgrade | nfs path
+  set_fact:
+   scale_nfs_url: 'ganesha_rpms/rhel8/'
+  when: ansible_distribution in scale_rhel_distribution and ansible_distribution_major_version == '8'
+
+- name: upgrade | nfs path
```

Comment:
The assignment of the path could be wrong when we allow the upgrade on lower releases as well. 
We set the path in other playbooks as follow. 

- name: install | nfs path
  set_fact:
   scale_nfs_url: 'ganesha_debs/ubuntu16/'
  when: ansible_distribution in scale_ubuntu_distribution and ansible_distribution_major_version != '20'

- name: install | nfs path
  set_fact:
   scale_nfs_url: 'ganesha_debs/ubuntu/'
  when: ansible_distribution in scale_ubuntu_distribution and ansible_distribution_major_version == '20'

Reply:
@Christop1964 Thank you. I am planning to add new files that you have changed. Thanks

Reply:
@Christop1964 Done